README.md

```diff
@@ -69,6 +69,7 @@ This attack is also known as Predictable Resource Location, File Enumeration, Di
     - [Docker Install](#docker-install)
 - [Configuration](#%EF%B8%8F-configuration)
     - [Default Values](#default-values)
+    - [Threads and Connection Limits At A High-Level](#threads-and-connection-limits)
```

Comment:
this link should be `[Threads and Connection Limits At A High-Level](#threads-and-connection-limits-at-a-high-level)` i believe